UKHomeOffice/cerberus-service
# COP-10939: Add tests to verify task with Crew members

## Description
Add tests to verify task with Crew members

## To Test
npm run cypress:runner

run the test `Should verify task summary displays Crew` from spec `airpax-task-overview-page.spec.js`

## Developer Checklist

\* Required
